src/main/java/org/openmrs/standalone/ApplicationController.java

```diff
@@ -262,14 +273,14 @@ public void run() {
 				unzipDatabase(new File("emptydatabase.zip"));
 				StandaloneUtil.resetConnectionPassword();
 				StandaloneUtil.startupDatabaseToCreateDefaultUser(mySqlPort);
-				Context.updateSearchIndex();
+			//	Context.updateSearchIndex();
 				System.out.println("Database mode using empty: " + applyDatabaseChange );
 			} else if (applyDatabaseChange == DatabaseMode.DEMO_DATABASE) {
 				deleteActiveDatabase();
 				unzipDatabase(new File("demodatabase.zip"));
 				StandaloneUtil.resetConnectionPassword();
 				StandaloneUtil.startupDatabaseToCreateDefaultUser(mySqlPort);
-				Context.updateSearchIndex();
+				//Context.updateSearchIndex();
```

Comment:
comment out these because they are crushing on Windows yet passing on linux and mac

Reply:
Then why not completely remove?

Reply:
okay but this means the search Index will be updated by loading the war itself

Reply:
I was referring to this: https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#Do-not-comment-out-code